jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/BuildImageMojoIntegrationTest.java

```diff
@@ -117,6 +132,8 @@ private static String buildAndRunAdditionalTag(
     verifier.executeGoals(Arrays.asList("clean", "compile", "jib:build"));
     verifier.verifyErrorFreeLog();
 
+    assertImageDigest(projectRoot);
```

Comment:
For the test, we can actually pull and build using the digest that was generated to assert that the image was actually pushed with that digest.